lookbook/docs/components/sub-header.md.erb

```diff
@@ -26,6 +26,7 @@ This component is an own implementation for OpenProject requirements of contextu
 
 - Restrict SubHeader actions to the page's content-specific actions (e.g. filtering or creating a new object).
 - Use IconButtons instead of Buttons for actions only when the icon is clear enough to describe the action.
+- Include all the actions to the SubHeader that adds or remove elements to the content bellow it (eg. New work package), and/or modifies the display of the content bellow (eg. Filter)
```

Comment:
Restrict SubHeader actions to the page's content-specific actions (e.g. filtering or creating a new object).

Reply:
Hi @HDinger sometimes ago @marcalcobe asks me to add this text here to the lookbook to clarify the sub header actions better.

Reply:
I think what I intended there was to specify something different between filtering and creating elements but I would be okay with only keeping the first sentene.